tracim/tracim/lib/utils.py

```diff
@@ -182,6 +183,13 @@ def get_rq_queue(queue_name: str= 'default') -> Queue:
         db=cfg.EMAIL_SENDER_REDIS_DB,
     ))
 
+def current_date_for_filename() -> str:
+    """
+    ISO8601 current date, adapted to be used in filename (for
+    webdav feature for example), with trouble-free characters.
```

Comment:
please give an example of the result. "trouble-free characters" give no information (except that "it should work"). Giving an example will help the user to understand

Reply:
Done.